chore: added PR template (#560)
contentful/contentful-management.js: .github/PULL_REQUEST_TEMPLATE.md

<!--
Thank you for opening a pull request.

Please fill in as much of the template below as you're able. Feel free to delete
any section you want to skip.
-->

## Summary

<!-- Give a short summary what your PR is introducing/fixing. -->

## Description

<!-- Describe your changes in detail -->

## Motivation and Context

<!--
Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
-->

## Checklist (check all before merging)

> - [ ] Both unit and integration tests are passed

Reviewer: ```suggestion
- [ ] Both unit and integration tests are passing
```

> - [ ] There are not breaking changes

Reviewer: ```suggestion
- [ ] There are no breaking changes
```

When adding a new method:

> - [ ] The new method is exported both thought default and plain CMA clients

Reviewer: ```suggestion
- [ ] The new method is exported through the default and plain CMA client
```

- [ ] All new public types are exported from `./lib/export-types.ts`

> - [ ] Added a unit for the new method

Reviewer: ```suggestion
- [ ] Added a unit test for the new method
```
